src/components/address-book/ImportDialog/index.tsx

```diff
@@ -70,15 +80,17 @@ const ImportDialog = ({ handleClose }: { handleClose: () => void }): ReactElemen
             setZoneHover(false)
           }}
           validator={abCsvReaderValidator}
-          onUploadRejected={(result: { file: File; errors?: Array<Error | string> }[]) => {
+          onUploadRejected={(result: { file: File; errors?: Array<Error | string | PapaparseErrorType> }[]) => {
             setZoneHover(false)
             setError(undefined)
 
             // csvReaderValidator error
             const error = result?.[0].errors?.pop()
 
             if (error) {
-              setError(error instanceof Error ? error.message : error.toString())
+              const errorDescription = error instanceof Error ? error.message : JSON.stringify(error)
```

Comment:
Stringify adds extra quotes around strings as you can see on Franco’s screenshot.